client/post-editor/media-modal/image-editor/image-editor-canvas.jsx

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable wpcalypso/i18n-no-variables */
```

Comment:
cc @aduth any chance we can tighten this up, or should we avoid `ctx.translate` calls?